### DISCUSSION TITLE
Docker?

### DISCUSSION THREAD

**Discussion by SauerDeschain** [vishnuraghavb/EnBizCard#19] [General]
This is fantastic and I'd like to install it via docker on my Synology. Are there any plans to set one up? Thank you.

**Comment by vishnuraghavb** [comment]
Hi @SauerDeschain. Thanks for your interest! You can now install EnBizCard using Docker. Please check the [self-hosting guide](https://github.com/vishnuraghavb/EnBizCard#self-hosting-guide---docker) on the repo.

**Reply by SauerDeschain** [reply]
Fantastic, thank you. I'm trying to install it via Synology Docker, and there isn't a docker for this on the docker hub. I'm trying to learn how to export it now from github but I'm still learning how to make all of this work. 

I found this guide: https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action

This may be beyond me but I'll give it a crack this weekend. I really appreciate the work you've done. Thank you.

**Reply by vishnuraghavb** [reply]
You can easily clone this repository to your Synology by installing Git then follow this [self-hosting guide](https://github.com/vishnuraghavb/EnBizCard#self-hosting-guide---docker) for the installation. And sorry, I have no desire to make EnBizCard available as a Docker image on Docker Hub.

**Comment by tj0xin** [comment]
Hi @vishnuraghavb,

I have setup a Physical server using docker.  and NGINX

but i want to make the index.vue page available at the domain using nginx.

How is that possible? 

I am assuming index.vue is the main page for creating vcards.
